modules/cudl-data-processing/lambda.tf

```diff
@@ -68,48 +68,48 @@ resource "aws_lambda_alias" "create-transform-lambda-alias" {
-  timeout       = var.db-lambda-information[count.index].timeout
-  memory_size   = var.db-lambda-information[count.index].memory
-  role          = aws_iam_role.assume-lambda-role.arn
-  layers        = [aws_lambda_layer_version.db-properties-layer.arn, var.datadog-layer-1-arn, var.datadog-layer-2-arn]
-  handler       = var.db-lambda-information[count.index].handler
-  publish       = true
-
-  vpc_config {
-    subnet_ids         = [data.aws_subnet.cudl_subnet.id]
-    security_group_ids = [data.aws_security_group.default.id]
-  }
-
-  file_system_config {
-    arn = aws_efs_access_point.efs-access-point.arn
-
-    # Local mount path inside the lambda function. Must start with '/mnt/', and must not end with /
-    local_mount_path = var.dst-efs-prefix
-  }
-
-  environment {
-    variables = var.lambda_environment_datadog_variables
-  }
-
-  depends_on = [aws_efs_mount_target.efs-mount-point]
-}
-
-resource "aws_lambda_alias" "create-db-lambda-alias" {
-  count = length(var.db-lambda-information)
-
-  name          = var.lambda-alias-name
-  function_name = aws_lambda_function.create-db-lambda-function[count.index].arn
-  #function_version = var.db-lambda-information[count.index].live_version
-  function_version = aws_lambda_function.create-db-lambda-function[count.index].version
-}
+# resource "aws_lambda_function" "create-db-lambda-function" {
+#   count = length(var.db-lambda-information)
+#
+#   function_name = substr("${var.environment}-${var.db-lambda-information[count.index].name}", 0, 64)
+#   description   = var.db-lambda-information[count.index].description
+#   s3_bucket     = var.lambda-jar-bucket
+#   s3_key        = var.db-lambda-information[count.index].jar_path
+#   runtime       = var.db-lambda-information[count.index].runtime
+#   timeout       = var.db-lambda-information[count.index].timeout
+#   memory_size   = var.db-lambda-information[count.index].memory
+#   role          = aws_iam_role.assume-lambda-role.arn
+#   layers        = [aws_lambda_layer_version.db-properties-layer.arn, var.datadog-layer-1-arn, var.datadog-layer-2-arn]
+#   handler       = var.db-lambda-information[count.index].handler
+#   publish       = true
+#
+#   vpc_config {
+#     subnet_ids         = [data.aws_subnet.cudl_subnet.id]
+#     security_group_ids = [data.aws_security_group.default.id]
+#   }
+#
+#   file_system_config {
+#     arn = aws_efs_access_point.efs-access-point.arn
+#
+#     # Local mount path inside the lambda function. Must start with '/mnt/', and must not end with /
+#     local_mount_path = var.dst-efs-prefix
+#   }
+#
+#   environment {
+#     variables = var.lambda_environment_datadog_variables
+#   }
+#
+#   depends_on = [aws_efs_mount_target.efs-mount-point]
+# }
+
+# resource "aws_lambda_alias" "create-db-lambda-alias" {
+#   count = length(var.db-lambda-information)
+#
+#   name          = var.lambda-alias-name
+#   function_name = aws_lambda_function.create-db-lambda-function[count.index].arn
```

Comment:
This refers to lines 71-112: This section could be removed as it will still exist in git history

Reply:
Removed.